docs/docs/evaluation/configuration-reference.md

```diff
@@ -0,0 +1,227 @@
+# Configuration Reference
+
+This page contains reference documentation for the configuration options for the MongoDB Chatbot Evaluation CLI.
+
+A Evaluation CLI config files is a CommonJS file that exports a `ConfigConstructor` function as its default export.
+
+For an example of setting up a configuration file, refer to the [Configuration](./index.md) documentation.
+
+To set up a configuration file, you must first install the following packages:
+
+```bash
+npm install mongodb-chatbot-evaluation
+```
+
+## API Reference
+
+For a full API reference of all modules exported by `mongodb-chatbot-evaluation`, refer to the [API Reference](../reference/evaluation/) documentation.
+
+This page links to the key reference documentation for configuring the Ingest CLI.
+
+## `ConfigConstructor`
+
+The [`ConfigConstructor`](../reference/evaluation/modules.md#configconstructor) function is the root configuration type for the Ingest CLI. This exports an [`EvalConfig`](../reference/evaluation/interfaces/EvalConfig.md) object.
+
+## Data Stores
+
+### `CommandMetadataStore`
+
+The [`CommandMetadataStore`](../reference/evaluation/interfaces/CommandMetadataStore.md) is an interface to interact with MongoDB collection that tracks metadata of each command run.
```

Comment:
Can we split this into two concepts?

- `CommandMetadataStore` is an interface for storing metadata every time a command is run
- We provide a constructor that implements `CommandMetadataStore` to store command metadata in MongoDB Atlas.

Same for the other stores

Reply:
updated